### PR TITLE
Main tab

### PR DESCRIPTION
**Pull request by zujkovicstasa**

### CODE REVIEW

File: cocktails9/cocktails9/MainTabView.swift [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+    var body: some View {
+        NavigationStack {
+            VStack {
+                Spacer()
+        
+                Group {
+                    switch selectedTab {
+                    case .cocktails:
+                        cocktailsView()
```
Comment (dunja-maksimovic-levi9-com) [inline cocktails9/cocktails9/MainTabView.swift]: Just capitalize all view names, in code and in file name

File: cocktails9/cocktails9/MainTabView.swift [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+        case favorites
+        case profile
+    }
+    
+    @State private var selectedTab: Tab = .cocktails
+
+    var body: some View {
+        NavigationStack {
+            VStack {
+                Spacer()
```
Comment (dunja-maksimovic-levi9-com) [inline cocktails9/cocktails9/MainTabView.swift]: It looks like you don't need spacer here

**Review by dunja-maksimovic-levi9-com — CHANGES_REQUESTED** [review]

File: cocktails9/cocktails9/Views/cocktailsView.swift [hunk 3]
```diff
@@ -0,0 +1,19 @@
+//
+//  cocktailsView.swift
+//  cocktails9
+//
+//  Created by Stasa Zujkovic on 28.11.24..
+//
+
+import SwiftUI
+
+struct cocktailsView: View {
```
Comment (n-markovic-levi9-com) [inline cocktails9/cocktails9/Views/cocktailsView.swift]:
```suggestion
struct CocktailsView: View {
```

File: cocktails9/cocktails9/Views/favoritesView.swift [hunk 4]
```diff
@@ -0,0 +1,18 @@
+//
+//  favoritesView.swift
+//  cocktails9
+//
+//  Created by Stasa Zujkovic on 28.11.24..
+//
+
+import SwiftUI
+
+struct favoritesView: View {
```
Comment (n-markovic-levi9-com) [inline cocktails9/cocktails9/Views/favoritesView.swift]:
```suggestion
struct FavoritesView: View {
```

File: cocktails9/cocktails9/Views/profileView.swift [hunk 5]
```diff
@@ -0,0 +1,19 @@
+//
+//  profileView.swift
+//  cocktails9
+//
+//  Created by Stasa Zujkovic on 28.11.24..
+//
+
+import SwiftUI
+
+struct profileView: View {
```
Comment (n-markovic-levi9-com) [inline cocktails9/cocktails9/Views/profileView.swift]:
```suggestion
struct ProfileView: View {
```

**Review by n-markovic-levi9-com — CHANGES_REQUESTED** [review]

**Comment by dunja-maksimovic-levi9-com** [comment]
Looks good, just rename file names as well to be capitalized. Tip for renaming things: right click/Refactor/Rename and it does it for you

**Review by dunja-maksimovic-levi9-com — APPROVED** [review]

**Review by n-markovic-levi9-com — APPROVED** [review]

**Pull request merged**